cl/users/utils.py

```diff
@@ -61,6 +61,7 @@ def convert_to_stub_account(user: User) -> User:
     """
     user.first_name = "Deleted"
     user.last_name = ""
+    user.is_active = False
```

Comment:
I think I caught a bug related to this change, maybe it's not common but I believe it's possible to happen to some users that follow the next actions:

- User delete their account from the delete page
- The same user decides to register again with the same email address in the future

The user won't be able to log in since their account was set `is_active` to `False` and this is not overridden when registering again.

Also, the user won't need to confirm their email address again because it was previously verified, might be necessary to set `profile.email_confirmed` to `False` when stubbing the account?

Reply:
I don't think this should be an issue. If there's an existing, inactive, stub account, it won't be used when somebody signs up with an account. Instead, they'll just create a new one. We allow more than one account with a single email address! Django is weird sometimes!